templates/awsebscsiprovisioner.yaml

```diff
@@ -19,7 +19,7 @@ spec:
   chartReference:
     chart: awsebscsiprovisioner
     repo: https://mesosphere.github.io/charts/stable
-    version: 0.2.9
+    version: 0.4.0
```

Comment:
i believe this is incorrect: https://github.com/mesosphere/charts/pull/122/files#diff-89a57bca70adf41ec198f6e05b535c57L9

Reply:
thanks @s12chung good catch!